### ISSUE TITLE
clojure-lsp - Add Missing Require

### ISSUE THREAD

**Issue by FongHou** [BetterThanTomorrow/calva#914]
Thank you @bpringe, @ericdallo for coming up the fix for clojure-lsp on Windows!

I just tried it, noticed a minor issue. Add Missing Require included metadata in require.

![image](https://user-images.githubusercontent.com/13973254/103393455-b6897400-4af0-11eb-83da-b754d58abce8.png)

**Comment by PEZ** [comment]
I have also seen this happen in dev, when I added the refactorings. But then it stopped behaving like that, and I thought I had found the right settings to prevent it...

**Comment by ericdallo** [comment]
Never saw this behaviour before  🤔

**Comment by bpringe** [comment]
https://github.com/clojure-lsp/clojure-lsp/issues/120 Here's a related issue but it looks like it was believed to be fixed. However, it must not be.

**Comment by ericdallo** [comment]
I could not reproduce these issues, do you have a sample where I can reproduce it?

**Comment by bpringe** [comment]
Might just be on Windows. I'll test when I can get to this, if no one else does before me.

**Comment by FongHou** [comment]
I can't reproduce with calva 2.0.143 + clojure-lsp 2020.12.31-17.22.22  on windows anymore. It's a low priority.

**Comment by PEZ** [comment]
It is not just on Windows. I've seen it on Mac. Now idea how to reproduce it though. But it is very probably upstream (clojure-lsp or beyond).

**Comment by ericdallo** [comment]
It can be an issue on clojure-lsp, but I have never seen that, and it's quite hard fix it without a sample to reproduce it.

**Comment by PEZ** [comment]
Indeed. I will update if I encounter this again. But as things stand now, none of us can't really do anything about it.

**Comment by bpringe** [comment]
Has anyone seen this in a while? If not, I think we can close this.
